sbnana/SBNAna/Cuts/NumuCuts.cxx

```diff
@@ -7,11 +7,11 @@ namespace ana{
 
   const Cut kNumuBasicQual([](const caf::SRSliceProxy* slc)
 			   { // 
-			     bool hastrk = ( slc->reco.ntrk > 0 );
+			     bool hastrk = ( slc->reco.npfp > 0 );
 			     if (!hastrk) return hastrk;
 
 			     unsigned int muIdx = (unsigned int)kPrimMuonIdx(slc);
- 			     double len = slc->reco.trk[muIdx].len;
+ 			     double len = slc->reco.pfp[muIdx].shw.len;
```

Comment:
Lots of this file has `trk` converted to `shw`

Reply:
Ah right. I'll go through and fix them. Thanks